tools/gen-bootstrap.sh

```diff
@@ -0,0 +1,117 @@
+#!/bin/bash
+
+set -eu -o pipefail
+
+# The script is designed for simplification of new bootstrap.snap generation.
+# Just run it in tarantool build directory after updating update.lua (which
+# introduces new data version) and examine its output.
+# The script will try to generate bootstrap.snap in the current directory, and
+# in case of success will offer to copy the file to the source tree.
+# The manual instruction by which the script was created is available here:
+MANUAL_LINK="https://github.com/tarantool/tarantool/wiki/How-to-generate-new-bootstrap-snapshot"
+# Now the script does the following:
+ACTIONS=( "rebuild tarantool.")
+ACTIONS+=("run box.cfg{} box.internal.bootstrap() by tarantool in a clean dir.")
+ACTIONS+=("take the latest snapshot file an rename it to bootstrap.snap file.")
+ACTIONS+=("edit the file and change line with VClock to an empty set: {}.")
+
+TNT_BINARY=./src/tarantool
+TEMP_DIR=./gen-bootstrap-tmp
+
+if ! "$TNT_BINARY" --version >/dev/null 2>&1; then
+    echo "Must be run from tnt build dir, with tarantool binary built in it."
+    exit 1
+fi
+
+if ! xxd -v >/dev/null 2>&1; then
+    echo "xxd must be installed."
+    exit 1
+fi
+
+if [[ -f "$TEMP_DIR" ]]; then
+    echo "Regular file $TEMP_DIR was found, has no idea what to do, exiting."
+    exit 1
+fi
+
+if [[ -d "$TEMP_DIR" ]]; then
+    echo "Folder $TEMP_DIR was found, it seems that previous run was failed."
+    read -p "Drop the folder and proceed [Yn]?" yn
+    case ${yn:-y} in
+        [Yy]* ) rm -r "$TEMP_DIR";;
+        [Nn]* ) echo "OK, exiting."; exit 0;;
+        * ) echo "Wrong answer."; exit 1;;
+    esac
+fi
+
+if ! cmake --build . -j; then
```

Comment:
This would better be conditional - rebuild only if a flag is set. The flag may be set to true by default though.